package/yast2.spec

```diff
@@ -186,26 +123,26 @@ installation with YaST2.
 %install
 %yast_install
 
-mkdir -p "$RPM_BUILD_ROOT"%{yast_clientdir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_desktopdir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_imagedir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_localedir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_moduledir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_scrconfdir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_ybindir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_ydatadir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_yncludedir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_libdir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_vardir}
-mkdir -p "$RPM_BUILD_ROOT"%{yast_vardir}/hooks
-mkdir -p "$RPM_BUILD_ROOT"%{yast_schemadir}/control/rnc
-mkdir -p "$RPM_BUILD_ROOT"%{yast_schemadir}/autoyast/rnc
-mkdir -p "$RPM_BUILD_ROOT"/etc/YaST2
+mkdir -p %{buildroot}%{yast_clientdir}
+mkdir -p %{buildroot}%{yast_desktopdir}
+mkdir -p %{buildroot}%{yast_imagedir}
+mkdir -p %{buildroot}%{yast_localedir}
+mkdir -p %{buildroot}%{yast_moduledir}
+mkdir -p %{buildroot}%{yast_scrconfdir}
+mkdir -p %{buildroot}%{yast_ybindir}
+mkdir -p %{buildroot}%{yast_ydatadir}
+mkdir -p %{buildroot}%{yast_yncludedir}
+mkdir -p %{buildroot}%{yast_libdir}
+mkdir -p %{buildroot}%{yast_vardir}
+mkdir -p %{buildroot}%{yast_vardir}/hooks
+mkdir -p %{buildroot}%{yast_schemadir}/control/rnc
+mkdir -p %{buildroot}%{yast_schemadir}/autoyast/rnc
+mkdir -p %{buildroot}%{_sysconfdir}/YaST2
```

Comment:
I think the directories are created automatically by `make install` (which is invoked from `yast_install`). I guess the reason it to have always created these directories even in case there is no file installed into that directory.